.github/workflows/ci.yml

```diff
@@ -81,108 +144,132 @@ jobs:
-      run: |
-         sudo mkdir -p /etc/pgagroal
-         sudo cp ../../doc/etc/*.conf /etc/pgagroal
-         ./pgagroal >> /dev/null 2>&1 &
-         pid=$!
-         sleep 5
-         ./pgagroal-cli ping
-      working-directory: /Users/runner/work/pgagroal/pgagroal/build/src/
-    - name: GCC/Stop pgagroal & postgres
-      run: |
-        ./pgagroal-cli shutdown
-        installed_pg=$(brew search postgresql | grep postgresql@ | tail -n 1)
-        brew services stop ${installed_pg}
-      working-directory: /Users/runner/work/pgagroal/pgagroal/build/src/
-    - name: rm -Rf
-      run: rm -Rf build/
-      working-directory: /Users/runner/work/pgagroal/pgagroal/
-    - name: Start postgres
-      run: |
-        installed_pg=$(brew search postgresql | grep postgresql@ | tail -n 1)
-        brew services start ${installed_pg}
-    - name: CLANG/mkdir
-      run: mkdir build
-      working-directory: /Users/runner/work/pgagroal/pgagroal/
-    - name: CLANG/cmake
-      run: export CC=/usr/bin/clang && export OPENSSL_ROOT_DIR=`brew --prefix openssl` && cmake -DCMAKE_BUILD_TYPE=Debug ..
-      working-directory: /Users/runner/work/pgagroal/pgagroal/build/
-    - name: CLANG/make
-      run: make
-      working-directory: /Users/runner/work/pgagroal/pgagroal/build/
-    - name: CLANG/Run pgagroal & confirm pgagroal is running
+        source /tmp/functions.sh
+        verify_shutdown
+      working-directory: /home/runner/work/pgagroal/pgagroal/build/src/
+    - name: CLANG/Run pgagroal as daemon with 'epoll' backend
       run: |
         sudo mkdir -p /etc/pgagroal
+        crudini --set ../../doc/etc/pgagroal.conf pgagroal log_type file
+        crudini --set ../../doc/etc/pgagroal.conf pgagroal log_path /dev/null
+        crudini --set ../../doc/etc/pgagroal.conf pgagroal ev_backend epoll
+        echo "host all all all trust" > ../../doc/etc/pgagroal_hba.conf
         sudo cp ../../doc/etc/*.conf /etc/pgagroal
-        ./pgagroal >> /dev/null 2>&1 &
-        pid=$!
+        ./pgagroal -d || exit
         sleep 5
-        ./pgagroal-cli ping
-      working-directory: /Users/runner/work/pgagroal/pgagroal/build/src/
-    - name: CLANG/Stop pgagroal & postgres
+      working-directory: /home/runner/work/pgagroal/pgagroal/build/src/
+    - name: CLANG/Run verify_running for 'epoll' backend
+      run: |
+        source /tmp/functions.sh
+        verify_running
+      working-directory: /home/runner/work/pgagroal/pgagroal/build/src/
+    - name: CLANG/Run verify_shutdown for 'epoll' backend
       run: |
-        ./pgagroal-cli shutdown
-        installed_pg=$(brew search postgresql | grep postgresql@ | tail -n 1)
-        brew services stop ${installed_pg}
-      working-directory: /Users/runner/work/pgagroal/pgagroal/build/src/
+        source /tmp/functions.sh
+        verify_shutdown
+      working-directory: /home/runner/work/pgagroal/pgagroal/build/src/
+
+
+
+      #  build-macos:
```

Comment:
Can't we test kqueue on Mac ? https://developer.apple.com/library/archive/documentation/System/Conceptual/ManPages_iPhoneOS/man2/kqueue.2.html